tests/fixtures.py

```diff
@@ -234,83 +234,81 @@ def extract_instruction_features(extractor, fh, bbh, ih) -> Dict[Feature, Set[Ad
 # note: too reduce the testing time it's recommended to reuse already existing test samples, if possible
 def get_data_path_by_name(name):
     if name == "mimikatz":
-        return os.path.join(CD, "data", "mimikatz.exe_")
+        return (CD / "data" / "mimikatz.exe_").as_posix()
```

Comment:
why return a str vs Path here?

Reply:
I was also confused with what to use. str(Path) gives string representation of Path and .as_posix is pathlib modules own method for string representations.

Reply:
lets do this then:
```suggestion
        return CD / "data" / "mimikatz.exe_"
```

Reply:
If the return is of type pathlib.Path then in next step we will need to change it back to string as after this it is passed to extractors like get_extractor / get_dnfile_extractor and `capa.features.extractors.dnfile.extractor.DnfileFeatureExtractor` or `capa.features.extractors.dotnetfile.DotnetFileFeatureExtractor` takes input path of type string only . the output of get_data_path_by_name is directly passed to extractors as strings.

Reply:
@williballenthin  , which method would be better for string representation of path `str(path)` or `path.as_posix()`

**str(path)**: It returns the platform-specific path string representation, using the appropriate path separator (\ on Windows and / on Unix-like systems).
**path.as_posix()**: It always uses the forward slash (/) as the path separator.

Reply:
can we update the "get extractor" routines to accept Path too? 

if we need to serialize to string only during runtime and the path is not saved anywhere then we should use "str". (we want to use the OS-specific format.) if we save the serialized string, such as in the result document report, then we should pick an explicit format (posix sounds good) and stick to it. that way we can pass those reports across operating systems.

Reply:
Thanks for clarifications.👍